crates/scheduler/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ### Minor
 
+- Migrate `or_trap!` to `or_fail!` for when there's no board support
```

Comment:
nit: `or_trap` and `or_fail` are implementation details, the changelog should be understandable from a user perspective (the PR will be automatically added during release for those who want to see the implementation)

```suggestion
- Return an error when calling an unimplemented API
```